src/messageParser.ts

```diff
@@ -0,0 +1,57 @@
+import * as Ably from 'ably';
+
+import { DefaultMessage, Message, MessageHeaders, MessageMetadata } from './Message.js';
+
+export interface MessagePayload {
```

Comment:
Are we using this outside this file? If not, we don't need to export it.

Reply:
No, I just missed it, will remove the export